mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/building/impl/PojoMapper.java

```diff
@@ -112,7 +115,8 @@ public PojoMapper(MappingBuildContext buildContext,
 			BeanReference<? extends IdentifierBridge<Object>> providedIdentifierBridge,
 			IdentityMappingMode containedEntityIdentityMappingMode,
 			TenancyMode tenancyMode, ReindexOnUpdate defaultReindexOnUpdate,
-			PojoMapperDelegate<MPBS> delegate) {
+			PojoMapperDelegate<MPBS> delegate,
+			BackendMappingHints hints) {
```

Comment:
I guess you could also... delegate... to `PojoMapperDelegate` when you need to retrieve the hints?
Or even have `PojoMapperDelegate` extend `BackendMapperContext` directly, if that makes sense.

Reply:
Hmm, I wanted to do so initially but `PojoMapperDelegate` seemed to have unrelated methods so I didn't want to put in the hints there, BUT if you are ok with it I'll do so 😄 .